elements/Base/Base.test.js

```diff
@@ -22,13 +22,12 @@ import Base from '.';
 const createComponent = TestUtils.makeCreateComponent(Base);
 
 describe('Base', () => {
-  let component, testRenderer;
+  let testRenderer;
 
   beforeEach(() => {
-    [component, testRenderer] = createComponent();
+    [, testRenderer] = createComponent();
```

Comment:
can we remove the comma here?

Reply:
If the comma is removed, `testRender` would be assigned the first returned value from `createComponent()` (what was previously `component`). The comma allows assigning the second value without needing to declare a variable for the first value. `component` wasn't being referenced anywhere in the test, so I removed it for linting. This syntax is a bit odd looking though, so I could also see just leaving it as is and have the linter ignore that line.